quickcheck-contractmodel/quickcheck-contractmodel.cabal

```diff
@@ -89,6 +89,6 @@ library
       quickcheck-threatmodel
     -- Plutus and Cardano dependencies
     build-depends:
-      cardano-api                                     >= 8.36 && < 8.37,
-      cardano-ledger-core                             >= 1.2.0 && < 1.10,
-      cardano-ledger-shelley                          >= 1.2.0 && < 1.9,
+      cardano-api                                     >= 8.38 && < 8.39,
```

Comment:
This is a little bit scary, we are trying to use `ThreatModel` with `hydra` - which depends on `8.37`. What's the versioning scheme of `cardano-api`?